common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1447,14 +1475,29 @@
       "The handle <handle> is invalid."
     ],
     "subClass" : {
-      "ALREADY_EXISTS" : {
```

Comment:
Was this a 3.5 only change? Just to make sure we're not changing something here.

Reply:
Yes, added in https://github.com/apache/spark/pull/42009/files#diff-74b78a86e87e47c520c1183be7ec8b5220378f123e47daad7f9e4cdd1a66336cR1386